include/dlaf/tune.h

```diff
@@ -63,16 +67,31 @@ struct TuneParameters {
   // - updateConfiguration in init.cpp to update the value from command line options and environment
   //   values
   // - getOptionsDescription to add a corresponding command line option
+
+  TuneParameters() {
+    // Some parameters require the pika runtime to be initialized since they depend on the number of
+    // threads used by the runtime. We initialize them separately in the constructor after checking that
+    // pika is initialized.
+
+    // TODO: Check upfront whether pika has been initialized to be able to give a better error message.
+    // If pika has not been initialized getting the default thread pool will either trigger an assertion
+    // or a segfault.
```

Comment:
I've opened an issue for this on the pika side: https://github.com/pika-org/pika/issues/808.